plugins/module_utils/network/awplus/argspec/acl/acl.py

```diff
@@ -0,0 +1,85 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2023 Allied Telesis
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The arg spec for the awplus_acl module
+"""
+
+
+class AclArgs(object):  # pylint: disable=R0903
+    """The arg spec for the awplus_acl module
+    """
+
+    def __init__(self, **kwargs):
+        pass
+
+    argument_spec = {'config': {'elements': 'dict',
+                     'options': {'acls': {'elements': 'dict',
+                                 'options': {'aces': {'elements': 'dict',
+                                                      'options': {'ICMP_type_number': {'type': 'int'},
+                                                                  'ace_ID': {'type': 'int'},
+                                                                  'action': {'choices': ['permit',
+                                                                                         'deny',
+                                                                                         'copy-to-cpu',
+                                                                                         'copy-to-mirror',
+                                                                                         'send-to-mirror',
+                                                                                         'send-to-cpu']},
+                                                                  'destination_addr': {'type': 'str'},
+                                                                  'destination_port_protocol': {'elements': 'dict',
+                                                                                                'options': {'eq': {'type': 'int'},
+                                                                                                            'gt': {'type': 'int'},
+                                                                                                            'lt': {'type': 'int'},
+                                                                                                            'ne': {'type': 'int'},
+                                                                                                            'range': {'options': {'end': {'type': 'int'},
```

Comment:
Does this need elements: dict like the source port protocol?

Reply:
Had a look in the model file, and I have forgotten to put 'elements: dict' in